### PR TITLE
Add confirmations for every option in the script

### PR DESCRIPTION
**Pull request by insolvency**
Added yes or no confirmations for most parts of the script, maintains the speed of the script whilst allowing most user customisation.

```ps1
try {
    Stop-Process -Name msedge -Force
} catch {
    Write-Output "Microsoft Edge is already shut"
}

try {
	Stop-Process -Name Teams -Force
} catch {
	Write-Output "Microsoft Teams is already shut"
}
```
This is the only part without a confirmation because I'm not sure what its for.

image

### CODE REVIEW

**Comment by Zoobdude** [comment]
Can we have a option at the beginning to run all?

**Comment by insolvency** [comment]
Not sure how to do that in a clean way
Tom wants me to make it input 11 "y"s into the terminal :(

**Comment by Zoobdude** [comment]
~~Maybe move all the options into functions? Then if user wants all, it runs all functions, if not, it goes through asking one by one?~~

File: main.ps1 [hunk 1]
```diff
@@ -11,39 +11,67 @@ catch {
...
 }
 
-UnPin-App "Microsoft Edge"
-UnPin-App "Microsoft Store"
-UnPin-App "Mail"
+function Confirmation{
+	param ([string]$text)
+
+	$option = Read-Host $text "[y/n]"
+	return $option -eq "y"
```
Comment (YummyBacon5) [inline main.ps1:25]: Could this be changed to case insensitivity check if the user has put "yes" or "y"?
And that if something other than "no" or "n" has been put, then ask the user to enter the input again?

**Review by YummyBacon5 — CHANGES_REQUESTED** [review]

**Comment by insolvency** [comment]
I added a confirmation override that might work

**Comment by Zoobdude** [comment]
I just had that idea (hence the strike through of last mssge), I like it.

File: main.ps1 (hunk 1 above)
Comment (Emorgan12) [reply main.ps1:25]: if people break it bc they are dumb that is their fault

**Review by Zoobdude — APPROVED** [review]

**Comment by YummyBacon5** [comment]
Bruh there were still issues

**Pull request merged**